### PR TITLE
Feature/multitenant api locationlog

### PR DESCRIPTION
**Pull request by satkunas**
# Description
Add locationlog to multitenant API
Add unit tests for implemented API endpoint

# Impacts
uses DAL

# Delete branch after merge
YES 

# NEWS file entries
## New Features
* Add locationlog to API

### CODE REVIEW

File: t/unittest/UnifiedApi/Controller/Locationlog.t [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+use Test::More tests => 4;
+use Test::Mojo;
+
+#This test will running last
+use Test::NoWarnings;
+my $t = Test::Mojo->new('pf::UnifiedApi');
+
+$t->get_ok('/api/v1/locationlog' => json => { })
```
Comment (julsemaan) [inline t/unittest/UnifiedApi/Controller/Locationlog.t]: Add some fake data here and validate the results in the test

**Comment by julsemaan** [comment]
Needs to have the spec updated

File: docs/api/spec/openapi.yaml [hunk 2]
```diff
@@ -541,6 +541,22 @@ paths:
...
+        not an empty value.
+      parameters:
+      - $ref: '#/components/parameters/cursor'
+      - *1
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UsersList'
```
Comment (julsemaan) [inline docs/api/spec/openapi.yaml]: You need to define the schemas for locationlog

File: lib/pf/UnifiedApi/Controller/Locationlog.pm [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+=cut
+
+use strict;
+use warnings;
+use Mojo::Base 'pf::UnifiedApi::Controller::Crud';
+use pf::dal::locationlog;
+
+has dal => 'pf::dal::locationlog';
+has id_key => 'locationlog_id';
+has resource_id => 'pid';
```
Comment (jrouzierinverse) [inline lib/pf/UnifiedApi/Controller/Locationlog.pm]: Needs to be the database primary key 'id'
Replace this
```perl
resource_id => 'pid';
```
With this
```perl
resource_id => 'id';
```

**Pull request merged**